MonoGame.Framework.Content.Pipeline/Graphics/MeshHelper.cs

```diff
@@ -95,9 +95,9 @@ public static void CalculateNormals(GeometryContent geom, bool overwriteExisting
                     // by Shuangshuang Jin, Robert R. Lewis, David West.
```

Comment:
Sorry for the delay.
It is not wrong, every channel has a length equal to the positionIndices (and VertexContent.Positions) collection (for a given geometry), which in turn is less or equal the actual positions (stored in MeshContent.Positions) once the mesh is finalized. It may be less (such as the case in the failing test) when between two geometries (same mesh) there is no full overlap of used vertices